src/core/public/http/http_setup.ts

```diff
@@ -176,7 +180,7 @@ export const setup = (
               const next = await interceptor.responseError(
```

Comment:
`HttpInterceptor`'s `responseError` handler is able to change the `request`, should we change this to match the behavior of `response` where it's unable to change the `request`?